packages/cyberstorm-forms/src/components/FormSelect.module.css

```diff
@@ -0,0 +1,64 @@
+.SelectTrigger {
+  display: inline-flex;
```

Comment:
Also generally speaking I feel this file doesn't adhere to all the style things we've set up so far and I'm not sure why you're actively creating code debt by ignoring them. I'm talking about variables for colors, sizes, spaces etc.